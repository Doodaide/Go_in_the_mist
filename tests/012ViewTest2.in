who
place M18
view
place P3
view
place E1
view
place B18
view
place O18
view
place A10
view
term