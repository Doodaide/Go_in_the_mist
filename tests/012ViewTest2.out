B
O18,xxxxxxxxxxxxxx........#..........................
C1,x......x......x......x......xxxxxxxxxxxxxxxxxxxxx
L3,..........................................xxxxxxx
L18,xxxxxxxxxxxxxx...........#.......................
P18,xxxxxxxxxxxxxx.......#.#.........................
M18,xxxxxxxxxxxxxx..........#.#......................
